BizHawk.Emulation.Cores/Consoles/WonderSwan/WonderSwan.IStatable.cs

```diff
@@ -39,25 +36,6 @@ private void SaveTextStateData(TextStateData d)
 			d.Frame = Frame;
 		}
 
-		public void SaveStateText(TextWriter writer)

```

Comment:
These are actual text states, no?  Looks like we're calling `BizSwan.bizswan_txtstatesave` as opposed to `BizSwan.bizswan_binstatesave`.  I don't think they should be removed.